zerver/openapi/zulip.yaml

```diff
@@ -10467,6 +10473,17 @@ paths:
                     **Changes**: New in Zulip 9.0 (feature level 244).
                   type: boolean
                   example: true
+                editable_by_user:
+                  description: |
+                    Whether an organization administrator has configured this profile field as
+                    editable by users.
+
+                    This field does not affect admin users, they are always able to edit profile
+                    fields.
```

Comment:
This language feels a bit off -- because the default is `true`, I feel like it's weird to talk about whether an administrator has configured it as such.

The same comment for `models.py` applies as well -- we should be clear that this applies to editing the value for their own account. And the admin sentence could be something like "Note that administrators can edit custom profile fields for any user regardless of this setting."

Reply:
This description is now:

    Whether regular users can edit this profile field on their own account.

    Note that organization administrators can edit custom profile fields for any user
    regardless of this setting.

    **Changes**: New in Zulip 9.0 (feature level 263).


Maybe this can be improved still.